nuget/helpers/lib/NuGetUpdater/NuGetUpdater.Core/Run/ApiModel/JobErrorBase.cs

```diff
@@ -30,6 +30,7 @@ public static JobErrorBase ErrorFromException(Exception ex, string jobId, string
             {
                 HttpStatusCode.Unauthorized or
                 HttpStatusCode.Forbidden => new PrivateSourceAuthenticationFailure(NuGetContext.GetPackageSourceUrls(currentDirectory)),
+                HttpStatusCode.TooManyRequests => new PrivateSourceBadResponse(NuGetContext.GetPackageSourceUrls(currentDirectory)),
```

Comment:
is it possible to retry these errors before abandoning them?

Reply:
I'll create an issue to track this but in the meantime I think this PR should still go in since it doesn't change the current behavior of failing in these cases 😄